### PR TITLE
[FIX] mrp, purchase, (sale_)stock: remove/update dead links

### PR DESCRIPTION
**Pull request by ticodoo**
Some documentation was removed to reduce technical debt + they deemed
were self-explainatory. The links to these pages are removed to match.

Some links were also updated/changed to redirect => these links have
been updated to match the new addresses.

Task: 2457087

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mrp/views/res_config_settings_views.xml [hunk 1]
```diff
@@ -18,7 +18,6 @@
                                 </div>
                                 <div class="o_setting_right_pane" id="workorder_settings">
                                     <label for="group_mrp_routings" string="Work Orders"/>
-                                    <a href="https://www.odoo.com/documentation/user/14.0/manufacturing/management/manufacturing_order.html#manage-manufacturing-with-routings-and-work-centers" title="Documentation" class="o_doc_link" target="_blank"></a>
```
Comment (Whenrow) [inline addons/mrp/views/res_config_settings_views.xml:21]: I would at least replace by this one https://www.odoo.com/documentation/user/14.0/manufacturing/management/manufacturing_order.html#manage-manufacturing-with-routings-and-work-centers

File: addons/purchase_stock/views/res_config_settings_views.xml [hunk 2]
```diff
@@ -17,7 +17,6 @@
 						</div>
 						<div class="o_setting_right_pane">
 							<label for="module_stock_dropshipping"/>
-							<a href="https://www.odoo.com/documentation/user/14.0/purchase/replenishment/flows/dropshipping.html" title="Documentation" class="o_doc_link" target="_blank"></a>
```
Comment (Whenrow) [inline addons/purchase_stock/views/res_config_settings_views.xml:20]: could replace by this one https://www.odoo.com/documentation/user/14.0/inventory/management/delivery/dropshipping.html

File: addons/mrp/views/res_config_settings_views.xml (hunk 1 above)
Comment (ticodoo) [reply addons/mrp/views/res_config_settings_views.xml:21]: Isn't that the dead link I just removed?

Comment (Whenrow) [reply addons/mrp/views/res_config_settings_views.xml:21]: haha yes wrong copy-paste. This is the good link :  https://www.odoo.com/documentation/user/14.0/manufacturing/management/bill_configuration.html#adding-a-routing

**Comment by ticodoo** [comment]
@Whenrow Updated the links. I bow to your elite documentation finding skills.

**Comment by Whenrow** [comment]
> @Whenrow Updated the links. I bow to your elite documentation finding skills.

Thanks :) It's more a "it would be very suspicious to not have this thing in the doc" skill. 
robodoo r+

**Pull request closed without merging**